docs/release-notes/release-notes.md

```diff
@@ -2,6 +2,14 @@
 
 This guide provides notes for major version releases. These notes may be helpful for users when upgrading from previous major versions.
 
+## v4.0.0
+
+### Node Operators (v4.0.0)
```

Comment:
Is that all the changes for node operators (also is there anything different in the operation of the multiplexer)? 

Are there any config changes for example

Reply:
The `grpc_laddr` should be set in the cometbft config.toml for running _with_ multiplexer

Reply:
Good catch. Added a note about rpc.grpc_laddr in https://github.com/celestiaorg/celestia-app/pull/4684/commits/8aca7a5b043298cc59ceda9f894be3cc1688aa15